upgrade/upgrade-scripts/verify-jar

```diff
@@ -54,20 +55,34 @@ function verify_jar_verify_cleanup() {
 		report_progress 100 "Upgrade verification was successful"
 	return "$rc"
 }
+#
+# use command line program because it supports long arguments.
+#
+TEMP=`getopt -a -o :df:l:o: --long fsCheckStartTs:,fsCheckEndTs:,containerInitStartTs:,containerInitEndTs: -n 'verify-jar' -- "$@"`
```

Comment:
I took a stab and cleaning up the option handling and usage printing code; see [here](https://github.com/prakashsurya/appliance-build/commit/70c4719ca60b67d98150d321cdee972cf3d5cad8)

IMO, with my changes, it's more readable and maintainable; let me know what you think. If you like it, you can easily pull it into your local tree with the following:
```
$ curl -s https://github.com/prakashsurya/appliance-build/commit/70c4719ca60b67d98150d321cdee972cf3d5cad8.patch | git am -
```
Then use the usual tools to squash it to a single commit, reset the author, etc.